sbag/generators/java/__init__.py

```diff
@@ -30,6 +32,21 @@ def get_type(prop):
     else:
         return 'entity'
 
+
+def get_association_type(prop):
+    """
+       Based on property type returns string saying if its base type, list or entity.
+       """
```

Comment:
This comment should be properly aligned and changed it's content appropriately to it's function.